ebuild-writing/using-eclasses/text.xml

```diff
@@ -25,6 +25,14 @@ illegal (except where the inheritance criteria are cache-constant <d/> see <uri
 link="::general-concepts/portage-cache"/>).
 </p>
 
+<p>
+When using <c>inherit</c>, sort the arguments (eclasses) alphabetically, except
+for where the phases exported by an eclass are affected by subsequent arguments,
```

Comment:
Could this be worded a little weaker, because I don't think is is imperative as a policy? For example "... it is good practice to sort the eclasses alphabetically ..."

Reply:
I've modified it but requested we make it [policy](https://bugs.gentoo.org/775686).